features/aanschrijfwijze-persoon-met-partner-zonder-adellijke-titel-predicaat.feature

```diff
@@ -0,0 +1,342 @@
+      | naam                  | waarde |
+      | geslachtsnaam (02.40) | .      |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992934                       |
+      | fields              | aanschrijfwijze                 |
+      Dan heeft de response een persoon met de volgende 'adressering' gegevens
+      | naam                 | waarde     |
+      | aanschrijfwijze.naam | F. Groenen |
+
+    Abstract Scenario: persoon met aanduiding naamgebruik "<naamgebruik>", heeft een partner zonder geslachtsnaam
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999992934 |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                           | waarde        |
+      | voornamen (02.10)              | Franklin      |
+      | geslachtsnaam (02.40)          | Groenen       |
+      | aanduiding naamgebruik (61.10) | <naamgebruik> |
+      En de persoon heeft een 'partner' met de volgende 'naam' gegevens
+      | naam              | waarde  |
+      | voornamen (02.10) | Jantine |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992934                       |
+      | fields              | aanschrijfwijze                 |
+      Dan heeft de response een persoon met een leeg 'adressering' object
+
+      Voorbeelden:
+      | naamgebruik |
+      | P           |
+      | V           |
+      | N           |
+
+    Scenario: persoon met aanduiding naamgebruik "E", heeft een partner met geslachtsnaam gelijk aan standaardwaarde
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999992934 |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                           | waarde   |
+      | voornamen (02.10)              | Franklin |
+      | geslachtsnaam (02.40)          | Groenen  |
+      | aanduiding naamgebruik (61.10) | E        |
+      En de persoon heeft een 'partner' met de volgende 'naam' gegevens
+      | naam              | waarde  |
+      | voornamen (02.10) | Jantine |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992934                       |
+      | fields              | aanschrijfwijze                 |
+      Dan heeft de response een persoon met de volgende 'adressering' gegevens
+      | naam                 | waarde     |
+      | aanschrijfwijze.naam | F. Groenen |
```

Comment:
er zitten nog geen rules en scenario's in voor de situatie dat er meerdere (ex) partners zijn en dus gekozen moet worden welke partner gebruikt moet worden in de aanschrijfwijze